tasks/responsive_images.js

```diff
@@ -313,11 +313,22 @@ module.exports = function(grunt) {
   var processImage = function(srcPath, dstPath, sizeOptions, tally, callback) {
     var image = gfxEngine(srcPath);
 
-    image.identify(function(err, data) {
+    image.identify("%m:%T:%s\n", function(err, dataRaw) {
       if(err){
         handleImageErrors(err, sizeOptions.engine);
       }
 
+      var lastLineData = dataRaw.trim().split("\n").slice(-1)[0].split(":");
```

Comment:
`\n` is explicitly used as a separator in line 316 so there is no need to use a platform specific line end